conferences/2016/csharp.yml

```diff
@@ -0,0 +1,38 @@
+---
+title:  "C#"
+short:  null
+start:  "18/11/2016 - 18:30"
+end:    "18/11/2016 - 21:00"
+respo:  "turfu"
+speakers:
+  - "Martin 'reaper' Leydier"
+  - "Gabriel 'Hazriel' Bertini"
+  # - "Yohann 'shepard' Léon" # Should have been a speaker but had to cancel
+  - "Meven 'mevouc' Courouble"
+abstract: >
+  "Conférence d'introduction au langage C# annuelle pour les Sups"
+event_form:
+  description:  "Conférence d'introduction au langage C# suivi d'un TP"
+  expected_public:
+    ionis:    180
+    members:  10
+    external: 0
+video:
+  streamed: False # Got technical problems so we were unable to stream :(
+  recorded: False # Actually it was recorded, but the sound was too bad :(
+location:
+  type: "Epita VJ"
+  room: "VA 208"
+handson:  True
+handson_details:
+  start:  "18/11/2016 - 21:00"
+  end:    "19/11/2016 - 06:00"
+  rooms:
+    - "VA 105" # VJN ?
+    - "VA 204" # VJN ?
```

Comment:
I do not recall which room the conference has been given. But I imagine if we used the VA 208, that means we did not use at all the VA 105 or 204 (which are better for conf.), maybe they were closed because of a JPO the day after or something...

Reply:
They were booked on our event form, that's why I included them here